invokeai/app/invocations/controlnet_image_processors.py

```diff
@@ -483,30 +485,67 @@ class TileResamplerProcessorInvocation(ImageProcessorInvocation):
 
     # res: int = InputField(default=512, ge=0, le=1024, description="The pixel resolution for each tile")
     down_sampling_rate: float = InputField(default=1.0, ge=1.0, le=8.0, description="Down sampling rate")
+    mode: Literal["regular", "blur", "var", "super"] = InputField(
```

Comment:
`"var"` seems to be unused. Should we remove it?

Reply:
I left in there to figure out if there's an additional use case for it. But even a regular resample seems to accomplish the same. I can get rid of it now.